cmd/wallet-web/src/pages/Signup.vue

```diff
@@ -99,6 +99,7 @@
                   <img class="inline-block object-contain mr-2 max-h-6" :src="provider.logoURL" />
                   <span id="signUpText" class="flex flex-wrap">{{ provider.signUpText }}</span>
                 </button>
+                <h1 hidden>{{ isLoggedIn }}</h1>
```

Comment:
why we need hidden field?

Reply:
computed value is not getting updated unless I use like this. Using this.isLoggedIn directly dont listen to updated value. please suggest if you know other way or i can look more later.

Reply:
add a TODO to fix it, since hidden field can be manipulated

Reply:
https://vuejs.org/v2/guide/computed.html#Watchers